tests/test_fault_interpretation.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+import uuid
+import resqpy.organize as rqo
+from resqpy.model import Model
+
+
+def test_is_equivalent_no_other(tmp_model):
+    #Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+        domain = "depth",
+        is_normal = True,
+        maximum_throw = 3,
+    )
+    #Act
+    result = fault_interp.is_equivalent(other = None)
+
+    #Assert
+    assert result is False
+
+
+def test_is_equivalent_other_not_FaultInterpretation(tmp_model):
+    # Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+        domain = "depth",
+        is_normal = True,
+        maximum_throw = 3,
+    )
+    # Act
+    result = fault_interp.is_equivalent(other = 55)
+
+    # Assert
+    assert result is False
+
+
+def test_is_equivalent_is_other(tmp_model):
+    # Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+    )
+    # Act
+    other = fault_interp
+    result = fault_interp.is_equivalent(other = other)
+
+    # Assert
+    assert result is True
+
+
+def test_is_equivalent_is_sameUUIDs(tmp_model):
+
+    # Arrange
+    object_uuid = uuid.uuid4()
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+    )
+    fault_interp.uuid = object_uuid
+
+    # Act
+    other = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation_other',
+    )
+    other.uuid = object_uuid
+    result = fault_interp.is_equivalent(other = other)
+
+    # Assert
+    assert result is True
+
+
+def test_is_equivalent_is_tectonic_boundary_one_none(tmp_model):
+
+    # Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+    )
+    fault_interp.tectonic_boundary_feature = 'something'
+
+    # Act
+    other = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation_other',
+    )
+    other.tectonic_boundary_feature = None
+    result = fault_interp.is_equivalent(other = other)
+
+    # Assert
+    assert result is False
+
+
+def test_is_equivalent_non_equivalent_tectonic_boundary(tmp_model):
+
+    # Arrange
+    tect_boundary = rqo.TectonicBoundaryFeature(tmp_model, kind = 'fault')
+
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+    )
+    fault_interp.tectonic_boundary_feature = tect_boundary
+
+    # Act
+    tect_boundary_other = rqo.TectonicBoundaryFeature(tmp_model, kind = 'fracture')
+
+    other = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation_other',
```

Comment:
Please change this title so that both titles are the same – that way we are sure it is the tectonic boundary feature that is causing the non-equivalent result.

Reply:
Done

Reply:
Nirjhor, Have you committed your latest changes? The titles are still showing as different when I view code changes in github.